docs/source/admin/upgrading.rst

```diff
@@ -63,11 +63,11 @@ Upgrade JupyterHub packages
 
 There are two environments where the ``jupyterhub`` package is installed:
 
-#. The *hub environment*, which is where the JupyterHub server process
+#. The *hub environment*: Which is where the JupyterHub server process
    runs. This is started with the ``jupyterhub`` command, and is what
    people generally think of as JupyterHub.
 
-#. The *notebook user environments*. This is where the user notebook
+#. The *notebook user environments*: This is where the user notebook
```

Comment:
Same here. If using a colon, drop `This is`

Reply:
I have made changes @minrk